index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+	<head>
+		<link rel="stylesheet" type="text/css" href="style.css">
+		<title>Project1</title>
+		<script src="https://kit.fontawesome.com/62fc5eef3c.js" crossorigin="anonymous"></script>
+	</head>
+	<body>
+		<!--Navbar -->
+		<header>
+			<nav>
+				<div class="nav-btns-left">
+					<p><a class="link" href="#about">About</a></p>
+					<p><a class="link" href="#portfolio">Portfolio</a></p>
+				</div>
+				<div class="nav-btn-right">
+						<p><a class="link" href="#contact">Contact</a></p>
+				</div>
```

Comment:
instead of having a inside p inside div, would it be possible here to just have the a tag?